src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -78,5 +83,36 @@ public void setGear(Gear gear) {
     public Gear getGear() {
         return currentGear;
     }
+
+    public void disableAutoShifting() {
+        autoShifting = false;
+    }
+
+    public void enableAutoShifting() {
+        autoShifting = true;
+    }
     
-}
+    public void shiftGears() {
```

Comment:
Add a doc comment explaining that this needs to be polled by the calling command, isn't obvious at first glance